atomicapp/constants.py

```diff
@@ -37,6 +37,7 @@
 ANSWERS_FILE_SAMPLE_FORMAT = 'ini'
 WORKDIR = ".workdir"
 LOCK_FILE = "/run/lock/atomicapp.lock"
+LOG_FILE = "/var/log/atomicapp.log"
```

Comment:
That /var/log inside the container? outside? Or.. `atomic` somehow manages it? `needs_doc`